concepts/observability-configurable-logging/07-multiple-logpipeline-investigation.md

```diff
@@ -0,0 +1,136 @@
+2. Two log pipelines:
+   - [Loki](./assets/logpipeline-invstigation/case-4/loki.yml)
+   - [Mock server](./assets/logpipeline-invstigation/case-4/mock-server.yml)
+
+![a](./assets/logpipeline-invstigation/case-4/case-4.svg)
+
+Result:
+- Mock server was down. The Loki output kept working.
+- The chunks in the `filesystem buffer` are rolled: Old chunks are discarded; new ones created.
+- When Loki output was down, the mock server output kept working.
+
+![mockserver-down](/assets/logpipeline-invstigation/case-4/dashboard-mock-down.png)
+
+
+![loki-down](/assets/logpipeline-invstigation/case-4/dashboard-loki-down.png)
+
+### Case 5
+Setup:
+1. One input with two outputs (Loki (Loki plugin), and mock server (http plugin)) without rewrite tags
+2. Two log pipelines:
+   - [Loki](./assets/logpipeline-invstigation/case-5/loki.yml)
+   - [Mock server](./assets/logpipeline-invstigation/case-5/mock-server.yml)
+
+![a](./assets/logpipeline-invstigation/case-5/case-5.svg)
+
+Result:
+- Mock server was down. This led to the filling of the tail filesystem buffer.
+- After the tail plugin filesystem buffer was full, it kept reading.
+- The chunks in the tail plugin filesystem buffer are rolled: Old chunks are discarded; new ones created.
+- Loki output stopped working as well.
+- The logs are stored in filesystem buffer before applying the kuberenets filter. There is a risk of not being able to fetch the kubernetes metadata of the logs. This applies if the pod has been terminated before the logs have been read from the filesystem again for next stage of pipeline processing.
+
+
+A known [Fluent Bit GitHub issue](https://github.com/fluent/fluent-bit/issues/4373) describes the same problem.
+
+### Case 6
+Setup:
+- Two Inputs and two outputs (Loki (grafana-loki plugin) and mock server (http plugin)) without rewrite tags
+- Two logpipelines: 
+  - [loki](./assets/logpipeline-invstigation/case-6/loki.yml)
+  - [mockserver](./assets/logpipeline-invstigation/case-6/mock-server.yml)
+![a](./assets/logpipeline-invstigation/case-6/case-6.svg)
+
+Result
+- Each pipeline has its own fileystem buffer.
+- When both outputs are stopped, both tail plugins are losing logs. They keep the latest logs, but the amount of logs is different.
+
+
+## Summary
+We performed various tests and found that with filesystem buffering (through rewrite_tag filter) is necessary to prevent loss of logs when one of the output is down. However loss of logs cannot be prevented completely, if the buffer is filled up then fluentbit keeps only the latest logs.
+
+Additionally following case wes tried
+1. With 2 input and 2 outputs and having `storage.max_chunks_pause on`. This option as mentioned in [documentation](https://docs.fluentbit.io/manual/administration/buffering-and-storage#input-section-configuration) did not apply to input filters. The fluentbit did not recongnize this option and the pod would not start. However applying to the service section did not have any affect.
+
+## Conclusion
```

Comment:
Maybe we could move the conclusion back to the other document to have only the technical description of the test cases here. That would make it one combined concept and not just a collection of unrelated files

Reply:
moved.